pushapkscript/test/test_googleplay.py

```diff
@@ -15,160 +17,179 @@
 @patch('pushapkscript.googleplay.push_apk')
 class GooglePlayTest(unittest.TestCase):
     def setUp(self):
```

Comment:
Note for the future: we should move this module to pytest (instead of unittest). Thanks to these changes, we can make `self.products` a fixture quite simply. This would also allow to parametrize some tests (like this one https://github.com/mozilla-releng/pushapkscript/pull/73/files#diff-b87bd96469c23de6c381f19acbdcbf18R116). Finally, it would just be consistent with the rest of *script.